cloudwatch_logs_common/package.xml

```diff
@@ -9,7 +9,7 @@
 
   <license>Apache 2.0</license>
 
-  <depend>aws_common</depend>
+  <depend version_lt='2.0.0'>aws_common</depend>
```

Comment:
Can you remove this `version_lt='2.0.0'` string as well?

Reply:
oh right, missed that one. done.